addOns/ascanrules/CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 ### Changed
 - The XML External Entity Attack scan rule now include example alert functionality for documentation generation purposes (Issue 6119).
 
+### Added
+- Sql injection rule unit tests for boolean based attacks
+
```

Comment:
Only changes affecting the user should be mentioned in the changelog, tests themselves not worth it.

Reply:
Removed